16/feoh/randomtitlepicker.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import random
+# You'll need to create a file named config.py containing a line like: apikey="<your_api_key>"
+from config import apikey
+
+# Random OMDB title picker.
+
+
+def generate_random_imdb_id():
+    # This is the highest IMDB title id at time of writing. This stinks. Wish I had a better way to
+    # dynamically set this!
+    max_imdb_title_number = 9916754
```

Comment:
use a constant at the top of the module?

Reply:
can you get this number from the API?

Reply:
One would think so, right? As near as I can tell the answer is no, which is kind of underwhelmng :)